documentation/next-generation-quickstart.md

```diff
@@ -78,7 +75,7 @@ Install the package
 As an example, to install the Azure Compute module, you would run :
 
 ```sh
-npm i @azure/arm-compute@30.0.0-beta.1
+npm i @azure/arm-compute@latest
```

Comment:
I don‘t think we need to add `@latest` here because it uses latest package by default.

Reply:
for those who's already install an older version of @azure/arm-compute in their project. npm will not pick up the latest one.